thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/pinot/summary/Cube.java

```diff
@@ -0,0 +1,91 @@
+package com.linkedin.thirdeye.client.pinot.summary;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import com.google.gson.Gson;
+
+
+// TODO: The cube should be queried from the backend database during runtime.
+public class Cube { // the cube (Ca|Cb)
+  private double ga; // sum of all ta
```

Comment:
I will give it a better name in the final version of the algorithm. For now, I am going to keep it in order to sync with the terms in the paper. I have improved the other parts of the code. Thanks for your comments. :-)